automock/boto.cfg

```diff
@@ -1,9 +1,13 @@
-[Credentials]
```

Comment:
Seems like we should kill this entire file now, boto3 only reads "Credentials" from a BOTO_CONFIG (which it considers an old boto2 config file).

Also if we're no longer able to configure our local dynamodb through this file our docker-compose setup will no longer work. I think that's a whole separate issue entirely from this PR, though

Reply:
Yeah, going to hardcode the credentials, since they're bogus anyway and we only need them for the test.